File: Data Structures & Algorithms/missing-number/submission-2.cs
public class Solution {
    public int MissingNumber(int[] nums) 
    {
        int n = nums.Length;
        int psum = (n * (n + 1 ))/2;
        int asum = 0;
        for(int i = 0; i < nums.Length; i++)
        {
            asum += nums[i];
        }   
        return psum - asum;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-2.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        visited = {}
        for i in range(len(nums)):
            d = target - nums[i]
            if d in visited:
                return [visited[d],i]
            else:
                visited[nums[i]] = i
        return []

